src/__tests__/resetpassword/ForgotPassword.test.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import moxios from "moxios";
+import Adapter from "enzyme-adapter-react-16";
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import { ForgotPassword } from "../../components/auth/ForgotPasswordComp";
+import { FORGOT_PWD } from "../../redux/action_types";
+import { forgotPassword } from "../../redux/actions/ForgotPasswordAction";
+
+Enzyme.configure({ adapter: new Adapter() });
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+//   configure Mock store
+const store = mockStore({ response: { message: "check your email" } });
+describe("user action", () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+  afterEach(() => {
+    moxios.uninstall();
+  });
+  const email = "test@gmail.com";
```

Comment:
Why is this here and not above `describe()`? Talking about const email

Reply:
I am moving it up.